stock_planning/models/stock_planning.py

```diff
@@ -144,8 +148,11 @@ def _get_required_increase(self):
     outgoing_qty = fields.Float(
         'Outgoing', compute=_get_product_info_location,
         digits_compute=dp.get_precision('Product Unit of Measure'))
-    incoming_to_date = fields.Float(
-        'Incoming to date', compute=_get_to_date,
+    move_incoming_to_date = fields.Float(
+        'Incoming to date from moves', compute=_get_to_date,
```

Comment:
La etiqueta debería ser "Incoming up to date" si lo que se quiere decir es "Entrante hasta la fecha"